lib/CMakeLists.txt

```diff
@@ -26,37 +27,18 @@ set(OPTSCHED_SRCS Scheduler/aco.cpp
   Wrapper/OptSchedMachineWrapper.cpp
   Wrapper/OptSchedDDGWrapperBasic.cpp
   Wrapper/OptSchedGenericTarget.cpp
-)
+  Wrapper/AMDGPU/GCNOptSched.cpp
+  Wrapper/AMDGPU/OptSchedGCNTarget.cpp
+  Wrapper/AMDGPU/OptSchedDDGWrapperGCN.cpp
 
-set(OPTSCHED_TARGET_DEPS LLVMCodeGen)
+  STATIC
```

Comment:
Don't use explicit library types. CMake lets you specify shared via `-DBUILD_SHARED_LIBS=ON` if you don't do this (I'm actually a little unsure whether `add_llvm_target()` messes with that, though)